build.sbt

```diff
@@ -28,5 +28,10 @@ lazy val root = (project in file("."))
   .aggregate(stags, cli)
   .settings(
     // Don't publish useless root artifacts
-    packagedArtifacts := Map.empty
+    packagedArtifacts := Map.empty,
+    //--prevent project publishing https://github.com/sbt/sbt/issues/313rf
```

Comment:
This link doesn't point to anything.

Reply:
Ok, I fixed the reference link on top of the re-based version in commit 6527a5d